docs/docs/data-structures/tree/events.mdx

```diff
@@ -5,9 +5,16 @@ sidebar_position: 5
 
 ## Whole-Tree Events
 
-The `TreeView` object exposes 2 events that communicate changes that affect the whole tree.
+The `TreeView` object exposes 2 events that communicate changes that affect the whole tree. For example:
```

Comment:
I think the section below flows a bit better, in how it introduces the events and then says "here's how you subscribe to them"

Reply:
Done